db/migrate/20181231151710_create_protocols_participant.rb

```diff
@@ -0,0 +1,26 @@
+class CreateProtocolsParticipant < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
This should be an `up` method. Because of the one-way data changes it will break on rollback.

Reply:
Created a new story for this optimization:  https://www.pivotaltracker.com/story/show/164875068